libnmstate/netapplier.py

```diff
@@ -48,5 +73,41 @@ def _apply_iface_admin_state(client, iface_state, nmdev):
         raise UnsupportedIfaceStateError(iface_state)
 
 
-class UnsupportedIfaceStateError(Exception):
-    pass
+def _index_by_name(ifaces_state):
+    return {iface['name']: iface for iface in ifaces_state}
+
+
+def _build_connection_profile(iface_desired_state):
+    CON_PROFILE_PREFIX = 'nms-'
```

Comment:
It is a good idea to discuss a naming scheme for profiles. IMHO just using the interface name as the profile name should suffice. Also we should maybe define some expectations about the systems that we are managing with nmstate, e.g. require that there is only one profile per device on the system, it is named after the device and it is limited to the device.

Reply:
In the design document, an option to target multiple system wide profiles is suggested.
We will probably not implement it any time soon (unless a scenario will present itself), but we should leave it as an option.
At the moment I have no objection to leaving it as you suggested, defining that the active/running profile has no prefix.
I will change it.